ezyrb/pod.py

```diff
@@ -1,14 +1,14 @@
 """
+Module for Proper Orthogonal Decomposition (POD) via default svd
```

Comment:
Actually the class proposes 3 different method for computing POD, not only SVD.
I would propose to change to avoid misleading documentation.